docs/faq.md

```diff
@@ -0,0 +1,72 @@
+# Frequently Asked Questions
+
+This page describes the most frequently asked questions about the Scorecard use, checks or relevance. These answers are continually changing. If you know any other question that could be interesting to be covered here, please [contribute](../CONTRIBUTING.md)!
```

Comment:
"This page answers frequently asked questions about Scorecards, including its purpose, usage, and checks. This page is continually updated. If you would like to add a question, please [contribute]!"